docs/pages/guides/using-supabase.mdx

```diff
@@ -0,0 +1,108 @@
+
+Supabase automatically [generates a REST API](https://supabase.com/docs/guides/api?utm_source=expo&utm_medium=referral&utm_term=expo-react-native) from your database and employs a concept called [row level security (RLS)](https://supabase.com/docs/guides/auth/row-level-security?utm_source=expo&utm_medium=referral&utm_term=expo-react-native) to secure your data, making it possible to directly interact with your database from your React Native application without needing to go through a server!
+
+Supabase provides a TypeScript client library called [supabase-js](https://supabase.com/docs/reference/javascript/introduction?utm_source=expo&utm_medium=referral&utm_term=expo-react-native) to interact with the REST API. Alternatively, Supabase also exposes a [GraphQL API](https://supabase.com/docs/guides/database/extensions/pg_graphql?utm_source=expo&utm_medium=referral&utm_term=expo-react-native) allowing you to use your favourite GraphQL client (e.g. [Apollo Client](https://supabase.github.io/pg_graphql/usage_with_apollo/)) should you wish to.
+
+## Prerequisites
+
+Head over to [database.new](https://database.new?utm_source=expo&utm_medium=referral&utm_term=expo-react-native) to create a new Supabase project.
+
+### Get the API Keys
+
+Get the Project URL and `anon` key from the API settings.
+
+1. Go to the [API Settings](https://supabase.com/dashboard/project/_/settings/api) page in the Dashboard.
+1. Find your Project `URL`, `anon`, and `service_role` keys on this page.
+
+## Using the Supabase TypeScript SDK
+
+Using [supabase-js](https://supabase.com/docs/reference/javascript/introduction?utm_source=expo&utm_medium=referral&utm_term=expo-react-native) is the most convenient way of leveraging the full power of the Supabase stack as it conveniently combines all the different services (database, auth, realtime, storage, edge functions) together.
+
+### Install and initialize the Supabase TypeScript SDK
+
+<Step label="1">
+#### Install supabase-js and dependencies
+
+After you have created your [Expo project](/get-started/create-a-project/), you can install supabase-js and dependencies using the following command:
+
+<Terminal cmd={['$ npx expo install @supabase/supabase-js @react-native-async-storage/async-storage react-native-elements react-native-url-polyfill']} />
+</Step>
+
+<Step label="2">
+#### Initialize supabase-js in your project
+
+Simply create a helper file to initialize the Supabase client.
+We need the API URL and the `anon` key that you copied [earlier](#get-the-api-keys).
+These variables are safe to expose in your Expo app since we have
+[Row Level Security](https://supabase.com/docs/guides/auth/row-level-security?utm_source=expo&utm_medium=referral&utm_term=expo-react-native) enabled on our Database.
+
+```js utils/supabase.ts
+import 'react-native-url-polyfill/auto'
```

Comment:
can you run prettier on this file? we have a prettierrc configured for docs https://github.com/expo/expo/blob/main/docs/.prettierrc

Reply:
I'm having trouble setting up the repo locally with the current airport wifi. I'll try again tonight.